src/MB_Toolbox.php

```diff
@@ -335,6 +335,16 @@ public function curlPOST($curlUrl, $post, $isAuth = FALSE) {
         )
       );
     }
+    elseif (strpos($curlURL, 'api.dosomething') !== FALSE) {
```

Comment:
Eh... but then it would kinda suck for you as a 3rd party user to have to be aware of the inner workings of the API. I think it's fine :+1:, but would be interested in knowing your thoughts on the above.